certbot/tests/test_util.py

```diff
@@ -111,3 +115,46 @@ def skip_unless(condition, reason):  # pragma: no cover
         return lambda cls: cls
     else:
         return lambda cls: None
+
+
+def make_lineage(self, testfile):
```

Comment:
With this method in `test_util`, does `DuplicativeCertsTest` have to inherit from `BaseRenewableCertTest`?